docs/protocol-design/introduction.md

```diff
@@ -49,6 +49,37 @@ Although there is cross-over between the two main areas of the living whitepaper
+
+2. High latency: Average confirmation times fluctuate between 10 and 300 minutes [^5]. In addition, most Bitcoin services require more than one confirmation before considering a transaction fully-settled [^6], which adds additional latency for end users.
+
+3. Power inefficient: The Bitcoin network consumes an estimated 67.26TWh per year (comparable to the power consumption of the Czech Republic), using an average of 570kWh per transaction [^7].
+
+Bitcoin, and other cryptocurrencies, function by achieving consensus on their global ledgers in order to verify legitimate transactions while resisting malicious actors. Bitcoin achieves consensus via an economic measure called Proof of Work (PoW). In a PoW system participants compete to compute a number, called a nonce, such that the hash of the entire block is in a target range. This valid range is inversely proportional to the cumulative computation power of the entire Bitcoin network in order to maintain a consistent average time taken to find a valid nonce. The finder of a valid nonce is then allowed to add the block to the blockchain; therefore, those who exhaust more computational resources to compute a nonce play a greater role in the state of the blockchain. PoW provides resistance against a Sybil attack, where an entity behaves as multiple entities to gain additional power in a decentralized system, and also greatly reduces race conditions that inherently exist while accessing a global data-structure. 
+
+An alternative consensus protocol, Proof of Stake (PoS), was first introduced by Peercoin in 2012 [^8]. In a PoS system, participants vote with a weight equivalent to the amount of wealth they possess in a given cryptocurrency. With this arrangement, those who have a greater financial investment are given more power and are inherently incentivized to maintain the honesty of the system or risk losing their investment. PoS does away with the wasteful computation power competition, only requiring light-weight software running on low power hardware. While Nano uses a weighted-voting system ([ORV](/protocol-design#orv-consensus)) that can be compared to PoS, it differs from traditional PoS because there is not one monolithic blockchain that requires leader selection (i.e. a staker or a miner) to extend, representatives do not create or produce shared blocks (groups of transactions), each Nano account has its own blockchain that only the owner can modify (representatives can only modify their own blockchains), transactions are evaluated individually and asynchronously (a Nano block is a single transaction, not a group of transactions), anyone can be a representative, no funds are staked or locked up, users can remotely re-delegate their voting weight to anyone at any time, representatives do not earn transaction fees, and representatives cannot reverse transactions that nodes have locally confirmed (due to [block cementing](/glossary#cementing)).
```

Comment:
I would split up this large paragraph, maybe starting a new one at "While Nano...". And perhaps try breaking out the changes mentioned with ORV over the others into a bulleted list, or something more formatted that is easier to digest.

Reply:
Split and converted to bullet, but also proposed a potential minimization rewrite that just links to the ORV page for the same details: https://github.com/nanocurrency/nano-docs/pull/382#issuecomment-699503996

Reply:
I like the minimized rewrite you referenced. Let's adjust to that and get it merged in.